README.md

```diff
@@ -55,72 +30,42 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 Hello Microverse <a name="about-project"></a>
 
-**[your_project__name]** is a...
+**Hello Microverse** is a simple project to start with Github Workflow and linter configurations.
 
 ## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
+> HTML, CSS, Github
```

Comment:
- [ ] This section should be structured like it was originally in the template. Simply replace the names with that names you you use. Example


```html
<details>
  <summary>Client</summary>
  <ul>
    <li><a href="">HTML</a></li>
   <li><a href="">CSS</a></li>   
  </ul>
</details>
```